package-lock.json

```diff
@@ -4656,6 +4656,16 @@
           "integrity": "sha512-bY6fj56OUQ0hU1KjFNDQuJFezqKdrAyFdIevADiqrWHwSlbmBNMHp5ak2f40Pm8JTFyM2mqxkG6ngkHO11f/lg==",
           "dev": true
         },
+        "ansi-styles": {
```

Comment:
why do we have this package-lock changes when nothing changes in package.json

Reply:
@tcho6319 have you tried to rerun `npm install`. maybe that will help to get rid of this change?

Reply:
@SamChou19815 I reran `npm install` and I can't seem to get rid of these changes